examples/audio/pretrain/emilia/path.sh

```
@@ -0,0 +1,60 @@
+cuda_prefix=/usr/local
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)